client/utils/calculatePosition.js

```diff
@@ -1,23 +1,30 @@
-export default function ({ newPosition, items, isFirstChild, insert = false }) {
+import insertActions from './insertActions';
```

Comment:
While you are in here, can you please document each scenario and rules being made

Reply:
Added `js-doc` for the function. IMO `js-doc` is sufficient in this case. If you disagree, I will add comments for `if` blocks, if that is what you asked in the original comment.

Reply:
The thing I would like to do then would be to use action type after ADD_AFTER cond (for else) to improve readability of when that rule is applied, if that makes sense. Also, insertActions should be renamed to InsertActions and have type def (and action string type replaced with that type)